obspy/clients/fdsn/client.py

```diff
@@ -198,10 +198,10 @@ def __init__(self, base_url="IRIS", major_versions=None, user=None,
         :type eida_token: str
         :param eida_token: Token for EIDA authentication mechanism, see
             http://geofon.gfz-potsdam.de/waveform/archive/auth/index.php. If a
-            token is provided, options ``user`` and ``password`` will be
-            overridden. This mechanism is only available on select EIDA nodes.
-            The token can be provided in form of the PGP message as a string,
-            or the filename of a local file with the PGP message in it.
+            token is provided, options ``user`` and ``password`` must be given.
```

Comment:
I think this should read `If a token is provided, options ``user`` and ``password`` must **not** be given.`